doc-content/enterprise-only/processes/process-definitions-con.adoc

```diff
@@ -2,9 +2,10 @@
 
 = Process definitions
 
-After you have created, configured, and deployed your project that includes your business processes, you can view the list of all the process definitions in *Menu* → *Manage* → *Process Definitions*. You can refresh the list of deployed process definitions at any time by clicking the refresh button in the upper-right corner. 
+After you have created, configured, and deployed your project that includes your business processes, you can view the list of all the process definitions in *Menu* → *Manage* → *Process Definitions*. You can refresh the list of deployed process definitions at any time by clicking the refresh button in the upper-right corner.
 
 The process definition list shows all the available process definitions that are deployed into the platform. Click on any of the process definitions listed to show the corresponding process definition details. This displays information about the process definition, such as if there is a sub-process associated with it, or how many users and groups exist in the process definition.
 
-In the process definition details section, you can click the *Diagram* tab to view associated BPMN2-based diagram of the process definition . You can also deploy a new process instance for the process definition by clicking the *New Process Instance* button in the upper-right corner.
+You can also define the default pagination option for all users under Manage (Process Definition, Process Instances, Tasks, Jobs and Execution Errors) and Task Inbox menu. For more information, see <<{enterprise-dir}/processes/admin-and-config/managing-business-central-process-administration-con.adoc#,Process administration>>.
 
+In the process definition details section, you can click the *Diagram* tab to view associated BPMN2-based diagram of the process definition . You can also deploy a new process instance for the process definition by clicking the *New Process Instance* button in the upper-right corner.
```

Comment:
standard terminology in jbpm is "start a new process instance" (instead of deploy).

Reply:
Changed as suggested.